tests/components/zwave_js/test_light.py

```diff
@@ -0,0 +1,377 @@
+"""Test the Z-Wave JS light platform."""
+from copy import deepcopy
+
+from zwave_js_server.event import Event
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    ATTR_MAX_MIREDS,
+    ATTR_MIN_MIREDS,
+    ATTR_RGB_COLOR,
+)
+from homeassistant.const import ATTR_SUPPORTED_FEATURES, STATE_OFF, STATE_ON
+
+BULB_6_MULTI_COLOR_LIGHT_ENTITY = "light.bulb_6_multi_color_current_value"
+
+
+async def test_light(hass, client, bulb_6_multi_color, integration):
+    """Test the light entity."""
+    node = bulb_6_multi_color
+    state = hass.states.get(BULB_6_MULTI_COLOR_LIGHT_ENTITY)
+
+    assert state
+    assert state.state == STATE_OFF
+    assert state.attributes[ATTR_MIN_MIREDS] == 153
+    assert state.attributes[ATTR_MAX_MIREDS] == 370
+    assert state.attributes[ATTR_SUPPORTED_FEATURES] == 51
+
+    # Test turning on
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": BULB_6_MULTI_COLOR_LIGHT_ENTITY},
+        blocking=True,
+    )
+
+    assert len(client.async_send_json_message.call_args_list) == 1
```

Comment:
We should add a fixture to mock commands/responses to make testing easier.

Reply:
I'd like to do this in a follow up.